action.yaml

```diff
@@ -57,17 +57,6 @@ inputs:
     description: Steps to run after auto-init / before check
     required: false
 
-  github-token:
-    description: For overriding github.token
-    required: false
-
-  trunk-token:
-    description:
-      You can find a per-repo API token in the Trunk web app settings. This will cause results to be
-      uploaded to the Trunk web app if this job is a scheduled job running on a branch, or if
-      `check-mode` is set to 'all'.
-    required: false
```

Comment:
We still want to support this path, for users with their own check upload workflow files, i.e. everyone currently using the web app for check uploads

Reply:
we don't have a way of masking these tokens

Reply:
Usually they're provided from `secrets` and get their own special masking